frontend/src/bundles/studio/pages/studio.tsx

```diff
@@ -28,6 +54,24 @@ const Studio: React.FC = () => {
                 }
             />
             <VideoMenu />
+            <VStack
+                sx={{
+                    position: 'fixed',
+                    left: '0',
+                    bottom: '0',
+                    width: '100%',
+                    height: '180px',
+                    alignItems: 'stretch',
+                }}
```

Comment:
Please, use css modules instead of sx prop 
See example https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/112

Reply:
Done) I had to add align-item directly to the stack because it was rewritten by chakra default style.